RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -736,27 +736,29 @@ void Event::createPixelTriplets() {
   // TODO: check if a map/reduction to just eligible pLSs would speed up the kernel
   // the current selection still leaves a significant fraction of unmatchable pLSs
   for (unsigned int i = 0; i < nInnerSegments; i++) {  // loop over # pLS
-    int8_t pixelType = pixelTypes[i];                  // Get pixel type for this pLS
+    int pixelType = pixelTypes[i];                     // Get pixel type for this pLS
```

Comment:
```suggestion
    PixelType pixelType = static_cast<PixelType>(pixelTypes[i]);                     // Get pixel type for this pLS
```
then there is no need for the casts below